index.html

```diff
@@ -0,0 +1,220 @@
+        <link rel="stylesheet" href="styles.css">
+        <title>Munana Clinton </title>
+     </head>
+     <body>
+        <header id="header">
+            <nav class="header-nav">
+                <a id="logo">Munana Clinton</a>
+                <div class="menu-bar">
+                    <span class="nav-bar"></span>
+                    <span class="nav-bar"></span>
+                    <span class="nav-bar"></span>
+                </div>
+                
+                </nav>
+        </header>
+        <main>
+        <section class="headline">
+            <div class="headline-content">
+                <h1 id="intro-text">Hey There. Iam Alex</h1>
+                <h2 id="description-text">I'm a Software Developer</h2>
+                <p id="welcome-text">I can help you build a product , feature or website 
+                    Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </p>  
+                <div class="my-socials">
+                    <a href="https://github.com/clin2on3mun"><img src="images/github.png" alt="github"></a>
+                    <a href="https://www.linkedin.com/in/munana-clinton-b6122720b/"><img src="images/linkedIn.png" alt="linkedIn"></a>
+                    <a><img src="images/angelist.png" alt="Angelist"></a>
+                    <a href="https://twitter.com/ClintonMunana"><img src="images/twitter.png" alt="twitter"></a>
+                    <a><img src="images/Medium.png" alt="Medium"></a>
+                         
+                </div>
+            </div>  
+        </section> 
+        <section id="work-section">
+                <section class="recent-work">
+                    <h2 id="recent-work-title">My Recent Works</h2>
+                    <hr>
+                    <img id="img-project" src="images/Img-Placeholder.png" alt="images-work">
+                    <h3 id="project-description-title">Multi-Post Stories</h3>
+                    <p id="project-description">A daily selection of privately personalized reads;
+                       no accounts or sign-ups required. has been the industry's standard dummy
+                       text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <div class="button-section">
+                       <ul class="buttons">
+
+                           <li><button class="btn-work">CSS</button></li>
+                           <li><button class="btn-work">HTML</button></li>
+                           <li><button class="btn-work">Bootstrap</button></li>
+                           <li><button class="btn-work">Ruby</button></li>
+                           <li><button class="view-button">See project</button></li>
+                        </ul>   
+                    </div>       
```

Comment:
i would suggest you use the HTML semantics tag (eg: section) where you use div you can read about<a href="https://www.pluralsight.com/guides/semantic-html">HTML semantics</a> to get an idea on changes and why you will make them